include/boost/math/special_functions/gamma.hpp

```diff
@@ -944,13 +972,15 @@ BOOST_MATH_GPU_ENABLED T full_igamma_prefix(T a, T z, const Policy& pol)
       {
          prefix = exp(alz - z);
       }
+      // LCOV_EXCL_STOP
    }
    //
    // This error handling isn't very good: it happens after the fact
    // rather than before it...
+   // Typically though this method is used when the result is small, we should probably not overflow here...
    //
-   if((boost::math::fpclassify)(prefix) == (int)BOOST_MATH_FP_INFINITE)
-      return policies::raise_overflow_error<T>("boost::math::detail::full_igamma_prefix<%1%>(%1%, %1%)", "Result of incomplete gamma function is too large to represent.", pol);
+   if((boost::math::fpclassify)(prefix) == (int)FP_INFINITE)
```

Comment:
```suggestion
   if((boost::math::fpclassify)(prefix) == (int)BOOST_MATH_FP_INFINITE)
```

Reply:
Good catch!

I'll do that before merging.

Unrelated question: should BOOST_MATH_FP_* map to the std values of FP_* ?